src/LibraryViewExtensionMSWebBrowser/ViewExtension.cs

```diff
@@ -26,7 +26,7 @@ public string UniqueId
             get { return "63cd0755-4a36-4670-ae89-b68e772633c4"; }
         }
 
-        public static readonly string ExtensionName = "LibraryUI2";
+        public static readonly string ExtensionName = "LibraryUI - MSWebBrowser";
```

Comment:
my reasoning for this funny naming was to avoid exposing the implementation in the name - not a big deal either way.

Reply:
I can revert this or do you have an alternative naming option?

Reply:
nah, it's fine - lgtm.